testing/ansible-tester/test-playbook.sh

```diff
@@ -159,9 +159,11 @@ setup_env() {
 
 wait_for_env() {
 	local inventory="$1"
-
-	printf 'waiting for environment to be ready\n'
```

Comment:
It's import to let the user know that they are waiting for the environment to be ready, so please don't delete this line.